driver_test.go

```diff
@@ -1314,8 +1319,18 @@ func TestArray(t *testing.T) {
 }
 
 func TestLargeSetResult(t *testing.T) {
+	CustomJSONDecoderEnabled = false
+	testLargeSetResult(t, 100000)
+}
+
+func TestLargeSetResultWithCustomJSONDecoder(t *testing.T) {
+	CustomJSONDecoderEnabled = true
+	// less number of rows to avoid Travis timeout
+	testLargeSetResult(t, 20000)
```

Comment:
a test case using the custom JSON decoder. The number of rows is 1/5 of the original one, because this is almost 4~5 times slower and hits timeout in Travis Test (10min)